### PR TITLE
Use dev-master for framework instead of tagged release

### PR DESCRIPTION
**Pull request by schlessera**
Fixes https://github.com/wp-cli/wp-cli/issues/4940

### CODE REVIEW

File: .behat-progress.log [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/Users/alain/dev/newcli/wp-cli-bundle/features/cli.feature:19
+/Users/alain/dev/newcli/wp-cli-bundle/features/cli.feature:31
+/Users/alain/dev/newcli/wp-cli-bundle/features/cli.feature:74
+/Users/alain/dev/newcli/wp-cli-bundle/features/cli.feature:103
+/Users/alain/dev/newcli/wp-cli-bundle/features/cli.feature:138
```
Comment (danielbachhuber) [inline .behat-progress.log]: @schlessera Was this meant to be committed?

**Comment by danielbachhuber** [comment]
@schlessera Should switching back to the stable release be documented in the release checklist?

**Comment by schlessera** [comment]
@danielbachhuber I don't think it is needed. The release will pull in the latest master from `wp-cli/wp-cli`, and at that moment in time, `dev-master` and the tagged release should point to the exact same revision.

**Comment by schlessera** [comment]
Regarding the `.behat-progress.log`, I think `*.log` should be added to all `.gitignore files to avoid this in the future.

**Comment by schlessera** [comment]
@danielbachhuber You're right, thinking about it some more, it does indeed matter outside of Phar files. I'll add a corresponding check to the check list.

**Pull request merged**